frontend/device/pocketbook/device.lua

```diff
@@ -67,6 +67,9 @@ local PocketBook = Generic:extend{
     -- Will be set appropriately at init
     isB288SoC = no,
 
+    -- Most of the devices are B&W.
+    hasColorScreen = no,
+
```

Comment:
This idea came to me too. But no success. Previously, I even applied it to certain PocketBook devices that I used for testing.